### ISSUE TITLE
VoiceOver navigation in vscode window may be trapped inside the top-left three-button region

### ISSUE THREAD

**Issue by yxycode17** [microsoft/vscode#93061]
Issue Type: <b>Bug</b>

1. On macOS enable VoiceOver screen reader.
2. Make sure that in the VoiceOver Utility app->Navigation page, Synchronize keyboard focus and VoiceOver cursor checkbox is checked.
3. Launch vscode. The VO cursor and keyboard focus should be at some UI element in the main window.
4. Navigate through the UI elements with VO command (keyboard or trackpad) till the VO cursor reaches the top-left three-button region (close, minimize and zoom buttons).  You will find that the keyboard focus remains at the web page, not synchronized with the VO cursor.
5. Then the problem occurs. When I try to navigate out of the button region and back to the web page below the title bar, I find I'm trapped. I have to restart VoiceOver to go back.

VS Code version: Code 1.43.1 (fe22a9645b44368865c0ba92e2fb881ff1afce94, 2020-03-18T07:01:41.240Z)
OS version: Darwin x64 19.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz (4 x 2700)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|8.00GB (0.15GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2020.3.69010
Go|ms-|0.13.1

</details>

**Comment by isidorn** [comment]
@yxycode17 hi, thanks for creating thi issue.
However this is a duplicate of https://github.com/microsoft/vscode/issues/88881
Let's continue the discussion there
